### DISCUSSION TITLE
Document why an address is in the blocklist?

### DISCUSSION THREAD

**Discussion by alphanet72** [Ultimate-Hosts-Blacklist/dev-center#65] [Q&A]
Is there a way to know the reason why an address is in the block list? 

If not, I understand you may not want to give the IP addresses of your honeypots, but did you think about at least putting a reason like (sends spam out, attacks SSH, hosts dangerous content, etc)?

Some kind of categorization.

Thank you.

**Comment by Wiggum127** [comment]
@alphanet72 My list contains comments highlighting the category why domains are blocked.

**Comment by alphanet72** [comment]
That's great. How may I reverse the info from https://github.com/Ultimate-Hosts-Blacklist/Ultimate.Hosts.Blacklist/blob/master/ips/ips0.list into your comments? :)

**Comment by Wiggum127** [comment]
I'm not sure I understand what you're suggesting. What are you suggesting to add and where?

**Comment by alphanet72** [comment]
Hello,

On Mon, Sep 02, 2024 at 06:52:18AM -0700, NHV wrote:
> I'm not sure I understand what you're suggesting. What are you suggesting to add and where?

When you look at the IP blocklist, how can you know the reason for one specific IP?

(I am not at all using your domain lists, only your IP block lists)

**Reply by spirillen** [reply]
You can in some cases find the info at @mypdns as this project primary goal is to be a index of this kind of info.

But as written, I've lost the full databases a couple of times, thanks to various reasons. This mean I only slowly rebuilding my index and would be appreciating if other others like @Wiggum127 would help filling in this info as he finds new resources, sharing his knowledge with the "Library"
